modules/multipath/data/org.freedesktop.UDisks2.Multipath.xml

```diff
@@ -0,0 +1,135 @@
+        corresponds to. It is '/' otherwise.
+    -->
+    <property name="Path" type="o" access="read"/>
+    <!-- Multipath:
+        If block is(or part of) multipath device, this is set to the
+        org.freedesktop.UDisks2.Multipath object that it corresponds to. It is
+        '/' otherwise.
+    -->
+    <property name="Multipath" type="o" access="read"/>
+  </interface>
+  <!--
+    org.freedesktop.UDisks2.Multipath:
+    @short_description: A multipath device.
+  -->
+  <interface name="org.freedesktop.UDisks2.Multipath">
+    <!-- WWID:
+        The WWID of certain multipath device(mpath).
+        It depends on "getuid_callout" configuration of multipathd.
+        Normally, it's SCSI VPD page 0x83 device ID or SCSI VPD page
+        0x80(rare) device ID of the disk device.
+    -->
+    <property name="WWID" type="s" access="read"/>
+    <!-- Name:
+        The alias of certain multipath device. For example: "mpatha".
+    -->
+    <property name="Name" type="s" access="read"/>
+    <!-- Block:
+        This is set to the org.freedesktop.UDisks2.Block object of
+        the /dev/dm-XX block
+    -->
+    <property name="Block" type="o" access="read"/>
+    <!-- Drive:
+        This is set to the org.freedesktop.UDisks2.Drive object.
+    -->
+    <property name="Drive" type="o" access="read"/>
+    <!-- Paths:
+        Array of org.freedesktop.UDisks2.Multipath.Path object paths.
+    -->
+    <property name="Paths" type="ao" access="read"/>
+  </interface>
+  <!--
+    org.freedesktop.UDisks2.Multipath.Path:
+    @short_description: A path of multipath path group device.
+  -->
+  <interface name="org.freedesktop.UDisks2.Multipath.Path">
+    <!-- Block:
+        This is set to the org.freedesktop.UDisks2.Block object.
+    -->
+    <property name="Block" type="o" access="read"/>
+    <!-- Status:
+        The status of this path provide by path checker. Should be one of
+        these values:
+            "UNKNOWN"
+            "UNCHECKED"
+            "DOWN"
+            "UP"
+            "SHAKY"
+            "GHOST"
+            "PENDING"
+            "TIMEOUT"
+            "REMOVED"
+            "DELAYED"
+        TODO: Explain here.
+    -->
+    <property name="Status" type="s" access="read"/>
+    <!-- Name:
+        The block name of this mpath path. Example: 'sdb' or 'nvme0n1'
+    -->
+    <property name="Name" type="s" access="read"/>
+    <!-- Multipath:
+        This is set to the org.freedesktop.UDisks2.Multipath object which
+        current path is belong to.
+    -->
+    <property name="Multipath" type="o" access="read"/>
+  </interface>
+  <!--
+    org.freedesktop.UDisks2.Manager.Multipath:
+    @short_description: Multipath add-on to the manager singleton
+  -->
+  <interface name="org.freedesktop.UDisks2.Manager.Multipath">
+    <!-- GetAllMultipaths:
+         Return an array of org.freedesktop.UDisks2.Multipath object paths
+         found on current system.
+    -->
+    <method name="GetAllMultipaths">
```

Comment:
When coding the test case for this new module, it found it's quite difficult to enumerate existing mpaths using python dbus, hence I introduced this method. I am open to better solution.